core/system/src/unvic.c

```diff
@@ -604,4 +569,12 @@ void unvic_init(void)
      * by IRQ number. For example, IRQ 0 has precedence over IRQ 1 if both have
      * the same priority level. */
     NVIC_SetPriorityGrouping(0);
+
+    /* Copy the user interrupt table into the handlers. */
```

Comment:
Could you add a comment here specifying the reason why this is needed and the consequences? In particular, it's important to specify that if a secure box code enables an IRQ before setting its ISR, the IRQ might be immediately served by the static ISR, which is out of the secure box control (because it's usually set by box 0/the host OS).

Related to this: https://github.com/ARMmbed/uvisor/pull/403#issuecomment-276931004.